src/components/text_input/text_input.ts

```diff
@@ -64,11 +64,44 @@ export class TextInput extends Component<Props, SpreadsheetChildEnv> {
     );
   }
 
+  onKeyDown(ev: KeyboardEvent) {
+    switch (ev.key) {
+      case "Enter":
+        this.save();
+        this.inputRef.el!.blur();
```

Comment:
this might dispatch twice no ? Once on `save` and once on `onBlur` that will call `save`